www/shared/styles/imports/variables.css

```
@@ -0,0 +1,72 @@
+@import "./custom-medias";
+
+/* ==========================================================================
+   Colors
+   ========================================================================== */
+
+:root {
+    --color-white: #fff;
+}
+
+/* ==========================================================================
+   Typography
+   ========================================================================== */
+
+:root {
+    --font-montserrat: "Montserrat";
+    --font-family: var(--font-montserrat), sans-serif;
+
+    --font-weight-regular: 400;
+    --font-weight-medium: 500;
+}
+
+/* ==========================================================================
+   Z-index
+   ========================================================================== */
+
+:root {
+    --z-index-base: 0;
+    --z-index-header: 2;
+}
+
+/* ==========================================================================
+   Transition / Animations timing
+   ========================================================================== */
+
+:root {
+    --duration-fast: 0.15s;
+    --duration-standard: 0.3s;
+    --duration-long: 0.4s;
+}
+
+/* ==========================================================================
+   Layout vars
+   ========================================================================== */
+:root {
```

Excuse me for my OCD:

```suggestion

:root {
```

---

I love your OCD ❤️

---

done!